src/L.Control.Locate.scss

```diff
@@ -15,3 +15,24 @@
   }
 }
 
+.leafet-control-locate-location circle {
+    animation: leaflet-control-locate-throb 4s ease infinite;
+}
+@keyframes leaflet-control-locate-throb {
+    0% { radius: 11; stroke-width:1; }
+   50% { radius:  9; stroke-width:3; }
+  100% { radius: 11; stroke-width:1; }
+}
+
+.leaflet-control-locate-circle {
```

Comment:
![crop](https://user-images.githubusercontent.com/589034/48104816-d7443080-e1e9-11e8-9c5b-629ccb085947.gif)

I mean, it's beautiful but also distracting.

Reply:
Looks like the NYT is using it on their map right now: https://www.nytimes.com/interactive/2018/11/05/opinion/election-midterms-2018-emotions.html

Reply:
In short because it does on ios maps app which I am emulating, and I think it adds value. Happy either way if you don't want it.

I was actually thinking of making it pulse only a single time and leaving it up to the redraw of the circle to kick off the new pulse so that this actually lets you know when the data is refreshed. This didn't work for some reason so I just left it as infinite. If we want to correctly handle a loss of signal then we probably need to handle that in a more robust way in code not css, eg a separate timeout which gets reset with each new location event, and with an alternate set of styles, ie the accuracy marker could be removed or it could turn gray or something. Either way this is beyond the scope of what I wanted to achieve here.

Reply:
Hmm, I don't see the effect on my ipad with apple maps.

Reply:
I'm on iphone ios 12. Just confirmed this doesn't show on my ipad either. I've just removed that and squished it back to one commit

Reply:
I watched some videos and they fade it out in the end.

Anyway, for prosperity, here is the CSS

```css
.leaflet-control-locate-circle {
  animation: leaflet-control-circle-pulse 5s ease infinite;
  transform-origin: center;
  transform-box: fill-box;
}

@keyframes leaflet-control-circle-pulse {
    0% { fill-opacity: .4;  transform: scale(.4); }
   20% { fill-opacity: .15; transform: scale(1);  }
   90% { opacity:  1; }
  100% { opacity: .1; }
}
```